lv_drv_conf.h

```diff
@@ -139,7 +139,7 @@
 #  define MONITOR_DOUBLE_BUFFERED 0
 
 /*Eclipse: <SDL2/SDL.h>    Visual Studio: <SDL.h>*/
-#  define MONITOR_SDL_INCLUDE_PATH    <SDL2/SDL.h>
+#  define MONITOR_SDL_INCLUDE_PATH    <SDL.h>
```

Comment:
Can we keep the original version if 
`-I/opt/homebrew/Cellar/sdl2/2.28.5/include` is used instead of 
`-I/opt/homebrew/Cellar/sdl2/2.28.5/include/SDL2`?

Reply:
It works . Nice call.